settings.gradle.kts

```diff
@@ -93,11 +87,10 @@ develocity {
   }
 }
 
-if (!useScansGradleCom) {
-  buildCache {
-    remote(develocity.buildCache) {
-      isPush = isCI && develocityAccessKey.isNotEmpty()
-    }
+buildCache {
+  remote(HttpBuildCache::class) {
+    url = uri("$develocityServer/cache/")
+    isPush = isCI && develocityAccessKey.isNotEmpty()
```

Comment:
this is the magic to get it to use our develocity server just for the (read-only) build cache 🤞